pyccl/nl_pt/lpt.py

```diff
@@ -0,0 +1,492 @@
+            pgg += ((bk21 + bk22)[:, None] * Pdmk2 +
+                    (bL12 * bk21 + bL11 * bk22)[:, None] * Pd1k2 +
+                    (b22 * bk21 + b21 * bk22)[:, None] * Pd2k2 +
+                    (bs2 * bk21 + bs1 * bk22)[:, None] * Ps2k2 +
+                    (bk21 * bk22)[:, None] * Pk2k2)
+
+        return pgg*self.exp_cutoff
+
+    def _get_pgm(self, trg):
+        """ Get the number counts - matter cross-spectrum at the internal
+        set of wavenumbers and scale factors.
+
+        Args:
+            trg (:class:`~pyccl.nl_pt.tracers.PTTracer`): number
+                counts tracer.
+
+        Returns:
+            array: 2D array of shape `(N_a, N_k)`, where `N_k` \
+                is the size of this object's `k_s` attribute, and \
+                `N_a` is the size of the object's `a_s` attribute.
+        """
+        self._check_init()
+        # Get biases
+        b1 = trg.b1(self.z_s)
+        b2 = trg.b2(self.z_s)
+        bs = trg.bs(self.z_s)
+        bk2 = trg.bk2(self.z_s)
+        b3nl = trg.b3nl(self.z_s)
+
+        # Compute Lagrangian bias
+        bL1 = b1 - 1
+        # Get Pk templates
+        if self.pk_b1 is None:
+            Pdmdm = self.lpt_table[:, :, 1]
+            Pdmd1 = 0.5*self.lpt_table[:, :, 2]
+            pgm = Pdmdm + bL1[:, None] * Pdmd1
+        else:
+            pgm = b1[:, None]*self.pk_b1
+        Pdmd2 = 0.5*self.lpt_table[:, :, 4]
+        Pdms2 = 0.25*self.lpt_table[:, :, 7]
+        Pdmo3 = 0.25 * self.lpt_table[:, :, 11]
+
+        if self.pk_bk is not None:
+            Pdmk2 = 0.5*self.pk_bk * (self.k_s**2)[None, :]
+        else:
+            Pdmdm = self.lpt_table[:, :, 1]
+
+            Pdmk2 = 0.5*Pdmdm * (self.k_s**2)[None, :]
+
+        pgm += (b2[:, None] * Pdmd2 +
+                bs[:, None] * Pdms2 +
+                b3nl[:, None] * Pdmo3 +
+                bk2[:, None] * Pdmk2)
+
+        return pgm*self.exp_cutoff
+
+    def _get_pmm(self):
+        """ Get the one-loop matter power spectrum.
+
+        Returns:
+            array: 2D array of shape `(N_a, N_k)`, where `N_k` \
+                is the size of this object's `k_s` attribute, and \
+                `N_a` is the size of the object's `a_s` attribute.
+        """
+        self._check_init()
+
+        pk = self.lpt_table[:, :, 1]
+
+        return pk*self.exp_cutoff
+
+    def get_biased_pk2d(self, tracer1, *, tracer2=None,
+                        extrap_order_lok=1, extrap_order_hik=2):
+        """Returns a :class:`~pyccl.pk2d.Pk2D` object containing
+        the PT power spectrum for two quantities defined by
+        two :class:`~pyccl.nl_pt.tracers.PTTracer` objects.
+
+        .. note:: The full non-linear model for the cross-correlation
+                  between number counts and intrinsic alignments is
+                  still work in progress in FastPT. As a workaround
+                  CCL assumes a non-linear treatment of IAs, but only
+                  linearly biased number counts.
```

Comment:
I think we can remove this note, since LPT can't compute any IA-related quantity at all.